web/modules/custom/dpl_article/dpl_article.info.yml

```
@@ -0,0 +1,5 @@
+name: "dpl_article"
+type: module
+description: "Functionality related to creating article content."
+package: "DPL"
+core_version_requirement: ^9 || ^10
```

```suggestion
core_version_requirement: ^10
```

Now that we are on Drupal 10 and do not expect to downgrade we might as well specify for the future.